pymediainfo/__init__.py

```diff
@@ -243,6 +243,7 @@ def parse(cls, filename, library_file=None, cover_data=False,
         :param bool full: display additional tags, including computer-readable values
             for sizes and durations.
         :param bool legacy_stream_display: display additional information about streams.
+        :param dict instance_options: set custom mediainfo options.
```

Comment:
I'd say something like
```
a dict containing additional options that will be passed to the `MediaInfo_Option` function."
```
and maybe add an example.

Reply:
Added your description with a value example.